torch/csrc/jit/script/compiler.cpp

```diff
@@ -1260,11 +1275,24 @@ struct to_ir {
 
     {
       pushFrame(body_block);
+      WithInsertPoint guard(body_block);
       if (itr_ident) {
+        if (in_list) {
+          // set user's iterator variable to the current element
+          auto listArg =
+              environment_stack->getVar(Var(max_trip_count.value()).name());
+          trip_count = emitBuiltinCall(
```

Comment:
Looks good! Using llvm style argument name comments (e.g.
```cpp
 1291: /*required=*/true);
```
) makes this big call easier to understand at first glance (and the clang-tidy CI job will verify the names actually match the function declaration)

Reply:
addressed